### PR TITLE
Review and integrate Time Conductor Vue style conversion

### PR DESCRIPTION
**Pull request by charlesh88**
Includes style and markup plus some @akhenry functional mods and fixes. **This should be squash merged.**

### Author Checklist

1. Changes address original issue? Y
1. Unit tests included and/or updated with changes? N/A
1. Command line build passes? N/A
1. Changes have been smoke-tested? Y

### CODE REVIEW

File: src/plugins/timeConductor/DatePicker.vue [hunk 1]
```diff
@@ -20,35 +20,107 @@
...
 <template>
-<span class="menus-up">
-    <a class="ui-symbol icon icon-calendar" ref="calendarButton" @click="togglePicker($event)"></a>
-    <div class="l-datetime-picker s-datetime-picker s-menu" v-if="showPicker" ref="popup">
-        <div class="holder">
-            <div class="l-month-year-pager">
-                <a class="pager prev" ng-click="changeMonth(-1)"></a>
-                <span class="val">{{model.month}} {{model.year}}</span>
-                <a class="pager next" ng-click="changeMonth(1)"></a>
+    <!-- TODOS: changeMonth doesn't appear to work, was ng-click -->
```
Comment (akhenry) [inline src/plugins/timeConductor/DatePicker.vue]: `changeMonth` should be working now.

File: src/plugins/timeConductor/ConductorModeIcon.vue [hunk 2]
```diff
@@ -0,0 +1,118 @@
+<template>
```
Comment (akhenry) [inline src/plugins/timeConductor/ConductorModeIcon.vue]: Needs copyright header

File: src/plugins/timeConductor/DatePicker.vue (hunk 1 above)
Comment (charlesh88) [reply src/plugins/timeConductor/DatePicker.vue]: Removed todo, confirmed is working.

File: src/plugins/timeConductor/ConductorModeIcon.vue (hunk 2 above)
Comment (charlesh88) [reply src/plugins/timeConductor/ConductorModeIcon.vue]: Sorry to miss this, added.

**Comment by akhenry** [comment]
## Reviewer Checklist

* Changes appear to address issue? Y
* Appropriate unit tests included? N/A
* Code style and in-line documentation are appropriate? Y
* Commit messages meet standards? Y

**Pull request merged**